### DISCUSSION TITLE
Feature request: API to list OAuth App policies for an organization

### DISCUSSION THREAD

**Discussion by XSchelin** [community/community#33178] [Apps, API and Webhooks]
There is already an API endpoint to get GitHub apps installed in an organization. I'd like there to be one to get third-party applications, as listed on the Integrations > Third-party access page in an org's settings.

**Comment by axox-eran** [comment]
Which endpoint gets you GitHub apps installed in an organization? I couldn't find one @XSchelin

**Comment by XSchelin** [comment]
https://docs.github.com/en/rest/apps/apps as far as I know.
